design/app-bar/toggle.module.css

```diff
@@ -2,11 +2,7 @@
   color: var(--gray-700);
 }
 
-.bar {
-  display: block;
-  width: 24px;
-  height: 4px;
-  margin-bottom: 4px;
-  background: currentColor;
-  border-radius: 1px;
+.icon {
+  width: 2rem;
+  height: 2rem;
```

Comment:
Do we need to resize the icon?

I'd prefer never resize it because it warps the image. Icons are drawn to be pixel perfect, they have 24x24 pixels size and each line is 2 pixels wide.